### PR TITLE
feature(api): add toolings end point to list supported terminologies

### PR DESCRIPTION
**Pull request by AAAlinaaa**

### CODE REVIEW

**Comment by apeteri** [comment]
Please add an example response JSON here.

File: core/com.b2international.snowowl.core.rest/src/com/b2international/snowowl/core/rest/admin/ConfiguredToolingsRestService.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+@Controller
+@RequestMapping(value = "/toolings") 
+public class ConfiguredToolingsRestService extends AbstractRestService {
+
+	@Operation(
+		summary = "Provides list of available tooling ids",
+		description = "Provides a set of all available toolings ids configured on the server."
+	)
+	@RequestMapping(method = { RequestMethod.GET }, produces = { AbstractRestService.JSON_MEDIA_TYPE })
+	public @ResponseBody Promise<Collection<Terminology>> toolings() {
```
Comment (cmark) [inline core/com.b2international.snowowl.core.rest/src/com/b2international/snowowl/core/rest/admin/ConfiguredToolingsRestService.java]: On REST interfaces, please always use `List` or `SortedSet` types to ensure consistent order.

Comment (cmark) [inline core/com.b2international.snowowl.core.rest/src/com/b2international/snowowl/core/rest/admin/ConfiguredToolingsRestService.java]: Promise wrapping is unnecessary.

**Review by cmark — CHANGES_REQUESTED** [review]

**Comment by AAAlinaaa** [comment]
[
  {
    "name": "Logical Observation Identifiers Names and Codes",
    "toolingId": "loinc",
    "effectiveTimeSupported": true,
    "terminologyComponents": [
      "com.b2international.snowowl.loinc.core.domain.LoincConcept"
    ],
    "dependencies": []
  }
]

**Comment by cmark** [comment]
> [ { "name": "Logical Observation Identifiers Names and Codes", "toolingId": "loinc", "effectiveTimeSupported": true, "terminologyComponents": [ "com.b2international.snowowl.loinc.core.domain.LoincConcept" ], "dependencies": [] } ]

That is too much information exposed from our internal tooling registry. The response should look like this:
```
{
  "toolings": [
    "snomed",
    "loinc",
    "icd10"
  ]
}
```

File: core/com.b2international.snowowl.core.rest/src/com/b2international/snowowl/core/rest/admin/ConfiguredToolingsRestService.java [hunk 2]
```diff
@@ -42,7 +43,11 @@ public class ConfiguredToolingsRestService extends AbstractRestService {
 		description = "Provides a set of all available toolings ids configured on the server."
 	)
 	@RequestMapping(method = { RequestMethod.GET }, produces = { AbstractRestService.JSON_MEDIA_TYPE })
-	public @ResponseBody List<Terminology> toolings() {
-		return TerminologyRegistry.INSTANCE.getConfiguredTerminologies();
+	public @ResponseBody ResponseEntity<List<String>> toolings() {
```
Comment (cmark) [inline core/com.b2international.snowowl.core.rest/src/com/b2international/snowowl/core/rest/admin/ConfiguredToolingsRestService.java]: No need for the ResponseEntity wrapping, but we need a wrapping object instead see the example JSON I have sent you.

**Review by cmark — CHANGES_REQUESTED** [review]

**Review by cmark — APPROVED** [review]
LGTM!

**Pull request merged**